Source/VMware.vSphereDSC/Tests/Integration/VMHostVss.Integration.Tests.ps1

```diff
@@ -200,42 +187,44 @@ try {
                 # Assert
-                { Get-DscConfiguration } | Should -Not -Throw
-            }
+                {
+                    $startDscConfigurationParameters = @{
+                        Path = $script:mofFileWithRemoveVss
+                        ComputerName = 'localhost'
+                        Wait = $true
+                        Force = $true
+                    }
+                    Start-DscConfiguration @startDscConfigurationParameters | Should -Not -Throw
+                }
 
-            It 'Should be able to call Get-DscConfiguration and all the parameters should match' {
-                # Act
-                $configuration = Get-DscConfiguration
+                It 'Should be able to call Get-DscConfiguration without throwing' {
+                    # Assert
+                    { Get-DscConfiguration } | Should -Not -Throw
+                }
 
-                # Assert
-                $configuration.Server | Should -Be $script:resourceWithRemoveVss.Server
-                $configuration.Name | Should -Be $script:resourceWithRemoveVss.Name
-                $configuration.Ensure | Should -Be $script:resourceWithRemoveVss.Ensure
-                $configuration.VssName | Should -Be $script:resourceWithRemoveVss.VssName
-                $configuration.NumPorts | Should -Be $script:resourceWithRemoveVss.NumPorts
-                $configuration.Mtu | Should -Be $script:resourceWithRemoveVss.Mtu
-                $configuration.Ensure | Should -Be $script:resourceWithRemoveVss.Ensure
-            }
+                It 'Should be able to call Get-DscConfiguration and all the parameters should match' {
+                    # Act
+                    $configuration = Get-DscConfiguration
+
+                    # Assert
+                    $configuration.Server | Should -Be $script:resourceWithRemoveVss.Server
+                    $configuration.Name | Should -Be $script:resourceWithRemoveVss.Name
+                    $configuration.Ensure | Should -Be $script:resourceWithRemoveVss.Ensure
+                    $configuration.VssName | Should -Be $script:resourceWithRemoveVss.VssName
+                    $configuration.NumPorts | Should -Be $script:resourceWithRemoveVss.NumPorts
+                    $configuration.Mtu | Should -Be $script:resourceWithRemoveVss.Mtu
+                    $configuration.Ensure | Should -Be $script:resourceWithRemoveVss.Ensure
+                }
 
-            It 'Should return $true when Test-DscConfiguration is run' {
-                # Arrange && Act && Assert
-                Test-DscConfiguration | Should -Be $true
+                It 'Should return $true when Test-DscConfiguration is run' {
+                    # Arrange && Act && Assert
+                    Test-DscConfiguration | Should -Be $true
+                }
             }
         }
     }
 }
 finally {
-}
+    Disconnect-VIServer -Server $Server -Confirm:$false
```

Comment:
This line is redundant as we are not connecting to any servers in these tests.

Reply:
Changed